README.md

```diff
@@ -1,37 +1,118 @@
-# SAS Viya Connector for ODPi Egeria
-
-## Overview
-
-This is the [SAS Viya](https://www.sas.com/sas-viya) Connector for ODPi Egeria, used to connect a SAS Information Catalog to the [ODPi/Egeria](https://github.com/odpi/egeria) ecosystem.  This is a low-level metadata exchange connector to connect with Egeria's [Open Metadata Repository Services (OMRS)](https://egeria.odpi.org/open-metadata-implementation/repository-services/)
-
-### What's New
-
-New project to connect to SAS Metadata.
-
-### Prerequisites
-
-ODPi/Egeria platform.
-
-### Installation
-
-COMING SOON 
-
-## Getting Started
-
-COMING SOON
-
-### Running
-
-COMING SOON
-
-### Troubleshooting
-
-**Optional**. Provide workarounds and solutions to known problems. Organize troubleshooting information using subtopics, as appropriate.
-
-## Contributing
-
-We welcome your contributions! Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to submit contributions to this project. 
-
-## License
-
-This project is licensed under the [Apache 2.0 License](LICENSE).
+# SAS-Egeria Connector
+
+## Overview
+The purpose of this project is to create an Egeria connector for SAS Information Catalog.
+
+Much of this code is based off of the Atlas-Egeria connector, which you can find [here](https://github.com/odpi/egeria-connector-hadoop-ecosystem/tree/master/apache-atlas-adapter).
+
+## Starting Egeria Server with Connector
+1. Run `gradlew clean build install` in this repository
+2. Bring up the docker containers in `src/test/resources/compose` with the command `docker-compose -f egeria-tutorial.yaml up`.<br/>
```

Comment:
Bear in mind the original compose images in egeria base do tend to change over time -- for example the UI recently. We're also discussing deprecating them in lieu of k8s. Obviously having your own here maintained in this project is fine :-)

Reply:
We removed those compose files, I will update the README.md.